iris_ugrid/tests/integration/test_ugrid_load.py

```diff
@@ -0,0 +1,69 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""
+Integration tests for the
+:mod:`iris.fileformats.ugrid_cf_reader.UGridCFReader` class.
+
+"""
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
+
+from gridded.pyugrid.ugrid import UGrid
+
+from iris.cube import CubeList
+from iris.fileformats.ugrid_cf_reader import CubeUgrid
+from iris import Constraint
+from iris.fileformats.netcdf import load_cubes
+
+
+@tests.skip_data
+class TestUgrid(tests.IrisTest):
+    def test_basic_load(self):
+        file_path = tests.get_data_path(
+            ("NetCDF", "unstructured_grid", "theta_nodal_xios.nc")
+        )
+
+        # cube = iris.load_cube(file_path, "theta")
+        # Note: cannot use iris.load, as merge does not yet preserve
+        # the cube 'ugrid' properties.
+
+        # Here's a thing that at least works.
+        loaded_cubes = CubeList(load_cubes(file_path, temp_xios_fix=True))
```

Comment:
The `temp_xios_fix` extension keyword needs to be removed here, as we're not using this now.
Cf #5 and https://github.com/SciTools-incubator/iris-ugrid/pull/4#discussion_r463710051